src/app/page.tsx

```diff
@@ -1,7 +1,13 @@
-export default function Home() {
+import {fetchPokemon} from "@/_api";
+
+export default async function Home() {
+    const {data} = await fetchPokemon()
   return (
     <main>
-      Its a next project!
+      <h1>All Pokemon</h1>
+        <ul>{data.pokemon_v2_pokemon.map(pokemon => {
+            return <div key={pokemon.id}>{pokemon.name}</div>
```

Comment:
If you set the key equal to pokemon.id, you'll only be able to add one of each type of pokemon to your bake. For example, if I wanted to add two bulbasaurs, their id would both be set to 1 and I would get an error. Consider changing the code to either have unique keys or prevent the user from choosing the same pokemon twice.